products/xcp-ng.md

```diff
@@ -0,0 +1,63 @@
+---
+title: XCP-ng
+category: os
+iconSlug: xcp-ng
+permalink: /xcp-ng
+alternate_urls:
+-   /xcpng
+versionCommand: software-version
+releasePolicyLink: https://xcp-ng.org/docs/releases.html#all-releases
+
+releases:
+-   releaseCycle: "8.2"
+    releaseDate: 2020-11-18
+    eol: false
+    support: 2025-06-25
+    lts: true
+    latest: "8.2.1"
+    
+releases:
```

Comment:
The `releases` attribute must not be repeated, or you will end up with only one release on the page (see the preview on https://deploy-preview-2445--endoflife-date.netlify.app/xcp-ng).

Reply:
Fixed.